src/CommunityToolkit.Maui/Alerts/Toast/Toast.windows.cs

```diff
@@ -0,0 +1,54 @@
+﻿using CommunityToolkit.WinUI.Notifications;
+using Windows.Data.Xml.Dom;
+using Windows.UI.Notifications;
+
+namespace CommunityToolkit.Maui.Alerts;
+
+public partial class Toast
+{
+	static ToastNotification? nativeToast;
+
```

Comment:
Since `nativeToast` is `static`, let's add and implement a `SemaphoreSlim(1,1)` like we did for `Snackbar.windows.cs` to ensure we don't induce any race conditions.

```suggestion
	static readonly SemaphoreSlim semaphoreSlim = new(1, 1);

```